ztp/source-crs/StorageLV.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "local-disks"
+  namespace: "openshift-local-storage" 
+spec:
+  logLevel: Normal
+  managementState: Managed
+  storageClassDevices:
+    - storageClassName: "fs-sc" 
+      volumeMode: Filesystem
+      fsType: xfs 
+      # The below must be adjusted to the hardware
+      devicePaths: 
+        - /dev/sdb
+---
```

Comment:
Please comment out this line as it causes issues with PolicyGen

Reply:
Done